server/src/http/feature_refresher.rs

```diff
@@ -22,6 +22,17 @@ use crate::{
     types::{ClientFeaturesRequest, ClientFeaturesResponse, EdgeToken, TokenRefresh},
 };
 
+fn frontend_token_is_covered_by_tokens(
+    frontend_token: &EdgeToken,
+    tokens_to_refresh: Arc<DashMap<String, TokenRefresh>>,
+) -> bool {
+    tokens_to_refresh.iter().any(|client_token| {
+        client_token
+            .token
+            .same_environment_and_broader_or_equal_project_access(frontend_token)
```

Comment:
This is the magic, this flips fe token and present tokens - we previously did the check the wrong way around